packages/webview_flutter/example/test_driver/webview_flutter_e2e.dart

```diff
@@ -828,6 +828,40 @@ void main() {
     final String currentUrl = await controller.currentUrl();
     expect(currentUrl, 'about:blank');
   });
+
+  testWidgets(
+    'can open new window and go back',
```

Comment:
this one is good, but what about a test case like the one in the internal doc?

Reply:
I'm not sure how to do a test like that. I would need to load a link that tries to open a window and test to see if javascript ran?